app/ApolloClientProvider.tsx

```diff
@@ -2,20 +2,20 @@
 
 import { ApolloLink, HttpLink } from '@apollo/client';
 import {
+  ApolloClient,
   ApolloNextAppProvider,
-  NextSSRApolloClient,
-  NextSSRInMemoryCache,
+  InMemoryCache,
   SSRMultipartLink,
-} from '@apollo/experimental-nextjs-app-support/ssr';
+} from '@apollo/experimental-nextjs-app-support';
```

Comment:
Imports are now directly on the `@apollo/experimental-nextjs-app-support` and no need for `/ssr` anymore